src/pages/what-we-do.pug

```diff
@@ -21,8 +21,8 @@ block pageComponents
   section.section
     .container
       +block({
-        blockHeading: "Insight",
-        blockContent: "<ul class=\"styled-list styled-list--inside-sm\"><li class=\"styled-list__item\"><strong>Behavioural analytics</strong> allow you to see what users are doing on your digital products, identifying sticking points based on actual user data.</li><li class=\"styled-list__item\"><strong>Split testing</strong> allows us to put multiple variants of a feature head to head to see which is best for your business, driving continual improvement.</li><li class=\"styled-list__item\"><strong>Staff shadowing</strong> gives us insight into any unnecessary overheads in your business processes, identifying opportunities for automation.</li></ul>",
+        blockHeading: "Research",
+        blockContent: "<p>First, we verify the problems you're having. We use techniques like <strong>staff shadowing</strong>, <strong>behavioural analytics</strong>, and <strong>recorded user sessions</strong> to validate any assumptions and establish a baseline for efficiency in the processes you're looking to improve.</p><p>From this, we'll put together a series of <strong>recommendations</strong> for using software to improve your level of automation, along with rationale on the benefits to your staff, customers, and bottom line.</p>",
```

Comment:
Added the comment here, although it spans the 6 new paragraphs on this page. 

Each paragraph should be independent of the other 5, but currently they have both a linear link ("First, "Finally"), and a paragraph link ("From this", "This allows us").

Taking _Research_ as the example, we could have something like this:

> We use techniques like **staff shadowing** and **behavioural analytics** to help uncover automation opportunities, and **recorded user sessions** to validate any assumptions and establish an efficiency baseline in the existing processes.
>
> We'll make a series of recommendations on how technology could automate both internal processes and external data-capture, along with rationale on the benefits to your staff, customers, and bottom line.